tracim/tracim/model/serializers.py

```diff
@@ -407,12 +408,14 @@ def serialize_node_for_page(content: Content, context: Context):
             })
         )
 
-        if content.type==ContentType.File:
+        if content.type == ContentType.File:
+            dpt = DepotManager.get()
```

Comment:
rename for making code more explicit:

dpt -> depot_manager
dpt_file -> depot_file

If I read the code and find `dpt_file`, I need to go where the var is setup. If the name is depot_file, I do not have to search for its setup: the name is explicit

Code is read 10 more times than it is written.